src/scripts/generateBridgeList.ts

```diff
@@ -66,13 +71,18 @@ export async function generateBridgedList(
     )
     .then((results) => parseBridgedAddressesResults(results, fetchBridgedAddressesCalls, tokensToReplace, chainId))
 
-  const tokens = await multicall.callStatic
+  const allTokens = await multicall.callStatic
     .tryAggregate(false, getTotalSupplyCalls(bridgedAddresses, provider))
     .then((results) => parseTotalSupplyResponses(results, bridgedAddresses, chainId))
 
+  const tokens = tokenFilter ? allTokens.filter(tokenFilter) : allTokens
```

Comment:
gist of the PR, the function receives now an optional parameter with the actual filter